### PR TITLE
haproxy logs that print the server IP

### PR DESCRIPTION
**Pull request by ramirogm**
Change-type: patch
Signed-off-by: Ramiro Gonzalez <ramiro.gonzalez@balena.io>

### CODE REVIEW

**Comment by ramirogm** [comment]
@balena-ci I self-certify!

File: apps/haproxy/haproxy.cfg [hunk 1]
```diff
@@ -70,7 +70,10 @@ frontend http
 		http-request capture req.hdr(Host) len 15
 		http-response lua.cors
 		# https://www.haproxy.com/blog/haproxy-log-customization/
-		log-format "%ci:%cp [%tr] %ft %b/%s %TR/%Tw/%Tc/%Tr/%Ta %ST %B %CC %CS %tsc %ac/%fc/%bc/%sc/%rc %sq/%bq %hr %hs %{+Q}r"
+		# https://www.haproxy.com/documentation/hapee/1-8r1/onepage/#8.2.4
+		# log-format "%ci:%cp [%tr] %ft %b/%s %TR/%Tw/%Tc/%Tr/%Ta %ST %B %CC %CS %tsc %ac/%fc/%bc/%sc/%rc %sq/%bq %hr %hs %{+Q}r"
```
Comment (joshbwlng) [inline apps/haproxy/haproxy.cfg:74]: @ramirogm Can we delete the old log formats to keep the files a bit cleaner? We can always see the previous log formats through git.

Comment (ramirogm) [reply apps/haproxy/haproxy.cfg]: Sure, no need to keep it

**Comment by ramirogm** [comment]
@balena-ci re-test

**Comment by ramirogm** [comment]
@balena-ci re-test

**Comment by ramirogm** [comment]
@balena-ci re-test

**Pull request closed without merging**